src/lib/is-config-page.ts

```diff
@@ -6,5 +6,5 @@
  */
 export function isConfigPage (hash: string): boolean {
   const isConfigHashPath = hash.startsWith('#/ipfs-sw-config') // needed for _redirects and IPFS hosted sw gateways
-  return isConfigHashPath
+  return isConfigHashPath && hash.endsWith('/ipfs-sw-config')
```

Comment:
we now have:

* **ipfs-sw-config**: show config page)
* **ipfs-sw-config-reload**: to trigger SW to pull new config from IDB and re-create verified-fetch)
* **ipfs-sw-config-get**: query the config object the SW last used to instantiate verified-fetch